locales/en-US/server.ftl

```diff
@@ -123,9 +126,17 @@ shotPageDownloadShot =
     .title = Download
 shotPageEditButton =
     .title = Edit this image
-shotPagefavoriteButton =
+shotPageFavoriteButton =
```

Comment:
This would require localizers to retranslate the string. If it's just for cosmetic reasons, I would avoid changing the ID.